nilearn/_utils/estimator_checks.py

```diff
@@ -369,7 +369,7 @@ def return_expected_failed_checks(
         if (
             not IS_SKLEARN_1_6_1_on_py_lt_3_13
             and SKLEARN_MINOR >= 5
-            and scipy_version != "1.8.0"
+            and scipy_version != "1.9.0"
```

Comment:
Should `scipy_version` be compared to `1.9.0` or this condition be removed? 

It seemed to me like, the below check should not be done with scipy `"1.8.0` but for greater versions it should stay.